scripts/main.js

```diff
@@ -71,4 +71,94 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// Opdracht 1 - Gegevens aanspreken
+
+console.log('De afdeling Sales heeft ' + departments.sales.numberOfEmployees + ' medewerkers');
+console.log('Marketing is een leuke afdeling om te werken. ' +departments.marketing.description);
+console.log('De afdeling Customer Service heeft ' +departments["customer-service"].numberOfEmployees + ' medewerkers');
+console.log('Sales is een uitdagende afdeling om te werken als Verkoopmanager.  ' +departments.sales.description);
+
+// Opdracht 2 - prompten en beslissen
+
+let gekozenAfdeling = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+
+if (gekozenAfdeling === 'marketing') {
+    console.log('Je koos marketing ' + departments.marketing.description);
+} else if (gekozenAfdeling === 'sales') {
+    console.log('Je koos sales ' + departments.sales.description);
+} else if (gekozenAfdeling === 'customer-service') {
+    console.log('Je koos customer-service ' + departments["customer-service"].description);
+}
+else {
+    console.error("Ongeldige keuze. Probeer het opnieuw door de pagina te verversen.");
+    }
+
+// Opdracht 3 - een nieuwe prompt
+
+if (gekozenAfdeling === 'marketing') {
+    let gekozenFunctieIndex = prompt(`Je koos marketing. Over welke functie wil je meer weten? Voer een getal tussen 0 en 4 in!
+    0: ${departments.marketing.jobs[0].title},
+    1: ${departments.marketing.jobs[1].title},
+    2: ${departments.marketing.jobs[2].title},
+    3: ${departments.marketing.jobs[3].title},
+    4: ${departments.marketing.jobs[4].title}`);
+
+}
+
+// console.log(input)
+//
+// if (input === '0') {
```

Comment:
Deze had je zelf ook al gezien, hier moeten we allebei de uitwerkingen even erbij pakken en prberen de switch/case en break-statements toe te passen.